src/Core/src/Platform/Windows/MauiWebView.cs

```diff
@@ -157,5 +170,21 @@ void SetupPlatformEvents()
 				}
 			};
 		}
+
+		static bool IsUriWithLocalScheme(string? uri)
+		{
+			return uri?
+				.StartsWith(
+					LocalScheme.TrimEnd('/'),
+					StringComparison.OrdinalIgnoreCase) == true;
+		}
+
+		static bool IsDataUri(string? uri)
+		{
+			return uri?
+				.StartsWith(
+					"data:text/html",
+					StringComparison.OrdinalIgnoreCase) ?? false;
```

Comment:
Not everyone is blessed with a widescreen and I don't mind the going deep not wide, we have it in other places I'm sure.

Reply:
Fitting the line on a GitHub diff on a mobile screen is one thing, but it's more about other devs taking the line and immediately understanding what the line does.

Reply:
Well, you're the reviewer, so it's the place to tell me, since you surely know the code base a lot better. I'll commit your suggestion.
And btw, I read code and diffs on mobile devices from time to time :-)